.travis.yml

```diff
@@ -2,6 +2,9 @@ language: go
 
 go_import_path: github.com/src-d/gitbase
 
+env:
```

Comment:
This might come from master instead as new line from this PR

Reply:
Yes, this file is copied from master so tests doesn't complain.